src/python/dxpy/scripts/dx.py

```diff
@@ -3596,10 +3634,9 @@ def run(args):
                 setattr(args, metadata, clone_desc.get(metadata))
 
         if clone_desc['class'] == 'job':
+            args.cloned_job_desc = clone_desc
             if args.executable == "":
                 args.executable = clone_desc.get("applet", clone_desc.get("app", ""))
-            args.input_from_clone = clone_desc["runInput"]
```

Comment:
`args.input_from_clone` is specified from `clone_desc` and used to override input as         ```exec_inputs.update(args.input_from_clone, strip_prefix=False)```

here I made it resolved later from `args.cloned_job_desc` as ```        exec_inputs.update(args.cloned_job_desc.get("runInput", {}), strip_prefix=False)```

@wormsik